gallery.js

```diff
@@ -2,12 +2,19 @@ let fs = require('fs');
 
 let imageFileNames = fs.readdirSync('site/images');
 
-// This code is just here to demonstrate how fs.readdirSync works
-
-console.log('sites/images contains the following files:')
+//This code is just here to demonstrate how fs.readdirSync works
 
+console.log('sites/images contains the following files:');
+let imageGallery;
```

Comment:
If you initialize this to `''` you don't need the conditional to check for `i === 0` down below.